images/build/php/7.4/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Build PHP 7.4
+FROM php-build-prereqs AS php74-build
+RUN set -eux && . /php/__phpVersions.sh && PHP_VERSION=$PHP74_VERSION GPG_KEYS=$PHP74_KEYS PHP_SHA256=$PHP74_TAR_SHA256 /php/build.sh
+
+# Install Composer so that it can be copied to the final build image later
```

Comment:
remove this section? since we discussed about building composer only once and we are currently using 7.3 version which is good enough

Reply:
ok